Library/Formula/git-credential-manager.rb

```diff
@@ -1,12 +1,13 @@
 class GitCredentialManager < Formula
   desc "Stores Git credentials for Visual Studio Team Services"
   homepage "https://java.visualstudio.com/Docs/tools/gitcredentialmanager"
-  url "https://github.com/Microsoft/Git-Credential-Manager-for-Mac-and-Linux/releases/download/git-credential-manager-1.4.0/git-credential-manager-1.4.0.jar"
-  sha256 "9a2c405b1876753734774fc63dc01ce8751d4af1a892bea59f823d996624efec"
+  url "https://github.com/Microsoft/Git-Credential-Manager-for-Mac-and-Linux/releases/download/git-credential-manager-1.5.0/git-credential-manager-1.5.0.jar"
+  sha256 "e5e685c6938e0955dabe91829d3bb4aaea9bce2bb1c6ecc6f405b74b4ce2b346"
 
   bottle :unneeded
 
-  depends_on :java => "1.7+"
+  depends_on :java => "1.8+" if MacOS.version >= :el_capitan
```

Comment:
You can make this a single block conditional rather than two:
```ruby
if MacOS.version >= :el_capitan
  depends_on :java => "1.8+"
else
  depends_on :java => "1.7+"
end
```